hail/src/main/scala/is/hail/expr/ir/lowering/IRState.scala

```diff
@@ -1,26 +1,27 @@
 package is.hail.expr.ir.lowering
 
+import is.hail.backend.ExecuteContext
 import is.hail.expr.ir.{
-  BaseIR, RelationalLet, RelationalRef, TableKeyBy, TableKeyByAndAggregate, TableOrderBy,
+  BaseIR, IRTraversal, RelationalLet, RelationalRef, TableKeyBy, TableKeyByAndAggregate,
+  TableOrderBy,
 }
 
-trait IRState {
+abstract class IRState(implicit E: sourcecode.Enclosing) {
```

Comment:
Need to pass context from the parent class otherwise the timings from `verify` won't include information about which condition is being verified`